src/main/java/Task.java

```diff
@@ -0,0 +1,56 @@
+public class Task {
+    protected static int length = 0;
+    protected String task;
+    protected boolean done;
+
+    public Task(String task, boolean done) {
+        this.task = task;
+        this.done = done;
+        length++;
+    }
+
+    public Task(String task) {
+        this.task = task;
+        this.done = false;
+        length++;
+    }
+
+    public Task() {
+        this.task = "";
+        this.done = false;
+    }
+
+    public String getDoneMarker() {
+        String doneMarker = "";
+        if (this.done) {
+            doneMarker = "[X]";
+        } else {
+            doneMarker = "[ ]";
+        }
+        return doneMarker;
+    }
```

Comment:
```suggestion
    public String getDoneMarker() {
        if (this.done) {
          return "[X]";
        }
        
        return "[ ]";
    }
```
Perhaps my suggestion can help to make your code more concise without creating an extra doneMarker variable?